### ISSUE TITLE
My collection of errors so far.

### ISSUE THREAD

**Issue by MCGSCEFFAAWHETJH55555** [Reperio-Studios/BrainBread-2-Issues#5]
Alright, so to clear this up.
My experience was done using multiplayer by myself since I was the first one to download and play it, it seems. As a player in Asia, I have about 210 ping at least out of all the servers.

-Time between shots and hit detection is pretty high, it feels like it is delayed.
-Some things can't be picked up. Things in question are the shotgun and the knife in laststand.
http://steamcommunity.com/sharedfiles/filedetails/?id=575987502
http://steamcommunity.com/sharedfiles/filedetails/?id=575987486
-I can't use this medkit on the wall in laststand, unsure if it's meant to be usable or not.
http://steamcommunity.com/sharedfiles/filedetails/?id=575987527
-Just as unsure if this is meant to be seen as ....'this' : http://steamcommunity.com/sharedfiles/filedetails/?id=575987516
-The bandits are very good at seeing you. As soon as an inch of you shows up in some opening, despite them not looking in that direction, they'll turn around and immediately start firing at you.
-Can't replenish ammo from dead NPC's guns if you have the same gun. Haven't tested if you have a different gun.
-Once you pick up armor or health from one spot, you can't pick it up from that same spot until you die and the next round starts.
-Zombies pay attention to breakable doors than you at times, even when they chase you. They also have their arms phasing through.
-Zombies are pretty good at hitting you, with no time to dodge. I'm not sure if it's just on my end but I find that if I dodge properly, the game prevents me from moving for a bit, it automatically slides me away once I successfully not get hit.
-The military start shooting you before you turn, but after you get infected. Not like they do damage, though.

**Comment by MCGSCEFFAAWHETJH55555** [comment]
Animation error for shotgun. Once you reload one shell, it reloads the subsequent shells by pulling the shells out of the magazine tube and then back inside.

**Comment by jonnyboy0719** [comment]
-  The pickup will be looked into and fixed up.
-  The time between shots is due to the lentency that the military is causing, will be fixed on the next update.
-  The medkit on the wall is not usable, only glowing items can be picked up.
-  you don't replenish ammo from ground weapons as of now (if you have the same weapon).
-  Armor & Health respawns after X amount of seconds, not directly. Its to prevent camping.
-  Zombie hitting you is something we really can't fix, its on the engine side, and we do not have access to the engine, and for the zombie that wants to make love to the doors when you stand near it, its being fixed and should be ready for the next update.

And for the shotgun animation, we know about this bug with the shotgun, but we are redoing the animations, so it should be fixed when it arrives. Just be sure you reload with 1 shell left, so it doesn't happen.

**Comment by MCGSCEFFAAWHETJH55555** [comment]
I know armor and health respawns, but once it respawns, you can't use it at that same spot.

**Comment by jonnyboy0719** [comment]
it randomly spawns a health kit or armor if not specified.

**Comment by MCGSCEFFAAWHETJH55555** [comment]
Yeah, but you can't use it at that same spot. You just can't after the first try on my end.

**Comment by jonnyboy0719** [comment]
You can still use the armor, if the armor is higher or lower (if you picked up a medium armor and trying to pickup light/heavy) you have to hold your _USE_ key down. We are working on a tutorial that will basically explain the basics of BrainBread 2, basically the important stuff.
